tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -4,6 +4,13 @@ xcodeproj(
     name = "Single-Static-Framework-Project",
     testonly = True,
     bazel_path = "bazelisk",
+    custom_build_options_by_config = {
+        "Release": [
+            "--define=BUILD_TYPE=release",
+            "--compilation_mode=dbg",
```

Comment:
should we be explicit about why this is set for release? (adding a comment here? cos to this can be unexpected)